ambry-clustermap/src/test/java/com.github.ambry.clustermap/AmbryReplicaSyncUpManagerTest.java

```diff
@@ -63,18 +63,23 @@ public AmbryReplicaSyncUpManagerTest() throws IOException {
     replicas.removeAll(localDcPeers);
     localDcPeerReplicas = new ArrayList<>(localDcPeers);
     remoteDcPeerReplicas = new ArrayList<>(replicas);
+    List<com.github.ambry.utils.TestUtils.ZkInfo> zkInfoList = new ArrayList<>();
```

Comment:
Please import static the ZkInfo class.

Reply:
Minor: since you are importing TestUtils.*, the code can just say ZkInfo not TestUtils.ZkInfo.